controller/filter.go

```diff
@@ -86,11 +86,11 @@ func (c *FilterController) List(ctx *app.ListFilterContext) error {
 		},
 		&app.Filters{
 			Attributes: &app.FilterAttributes{
-				Title:       "Workitem type",
+				Title:       "Type",
```

Comment:
This is the only change you should do. The rest should remain the same because it touches the implementation. The `Title` is only for display in the UI.

Reply:
Thought so too.
Thanks, updating!